src/utils/cacheHandlers.ts

```diff
@@ -30,6 +30,9 @@ export const fetchCachedResponse = async (request: any): Promise<any> => {
   if (cacheKey === null) {
     return request;
   }
+  if (request.cancelToken && request.setCancelTokenCacheKey) {
```

Comment:
I think we will have to add a comment here, as it might be pretty confusing why this is here if the dev isn't aware of the bug.

Reply:
Some comment was added, but whole thing is still confusing